__tests__/api.test.js

```diff
@@ -97,3 +98,13 @@ describe("PATCH /api/articles/:article_id", () => {
     expect(body.msg).toBe("Not Found");
   });
 });
+
+describe("GET /api/users", () => {
+  test("Returns array of user objects *which contain only the username*", async () => {
+    const { body } = await rerquest(app).get("/api/users").expect(200);
+    expect(body.users.length > 0).toBe(true);
+    users.forEach((user) => {
```

Comment:
instead of requiring in and using this data, try and get the information from the body!